lib/module.js

```diff
@@ -14,7 +14,8 @@ export default async function (moduleOptions) {
     cookie: {
       key: 'nuxt-color-mode',
       options: {
-        path: this.options.router.base
+        path: this.options.router.base,
+        sameSite: 'Lax'
```

Comment:
Is there a reason not to default to 'strict'?

Reply:
This looks like it has good description of the differences: https://web.dev/samesite-cookies-explained/

(about `strict` value)
> When the user is on your site, then the cookie will be sent with the request as expected. However when following a link into your site, say from another site or via an email from a friend, on that initial request the cookie will not be sent.

Reply:
As I wrote in the description, I'm not confident enough to take that decision on my own. So I leaned on the more lenient alternative 'lax'. 

If 'strict' wouldn't affect users negatively for sure, then it should definitely be set to 'strict'.